freshmaker/parsers/internal/manual_rebuild.py

```diff
@@ -51,10 +52,16 @@ def parse_post_data(self, data):
             errata = Errata()
             advisory = ErrataAdvisory.from_advisory_id(errata, errata_id)
 
-            event = ManualRebuildWithAdvisoryEvent(
-                msg_id, advisory, data.get("container_images", []), data.get("metadata", None),
-                freshmaker_event_id=data.get('freshmaker_event_id'), manual=True,
-                dry_run=dry_run, requester=data.get('requester', None))
+            # If we want to manually retrigger bundle rebuild, create
+            # BotasErrataShippedEvent event
+            if advisory.state == "SHIPPED_LIVE" and \
+                    advisory.reporter.startswith('botas'):
+                event = BotasErrataShippedEvent(msg_id, advisory)
```

Comment:
I believe we need different event to distinguish the original `BotasErrataShippedEvent` and the re-triggered one, so reusing `BotasErrataShippedEvent` is not what we want.